pages/involved_lessons.md

```diff
@@ -4,50 +4,41 @@ title: "Development of Lessons"
-For suggestions, and to propose large-scale changes,
-please start a conversation through an issue rather than putting in a pull request.
-This gives the lesson Maintainers and the rest of the
-community an opportunity to discuss the suggestion before you've done all of the
-work. If you're not familiar with GitHub, you can
-also submit a comment or question through one of our [communication channels][comms].
+## Suggesting Improvements
+
+HPC Carpentry lessons are collaboratively developed: anyone can contribute and
+we receive contributions regularly.
+All of our lessons are on GitHub and [creating an account][join-github] is
+required to contribute directly to the lessons.
+To fix a typo, correct an error, clarify an example, or otherwise suggest an
+improvement to an existing lesson, you can file an issue on the lesson's GitHub
+repository.
+For typo fixes and error corrections, pull requests are also welcome!
+For suggestions, and to propose large-scale changes, please start a
+conversation through an issue rather than putting in a pull request.
+This gives the lesson Maintainers and the rest of the community an opportunity
+to discuss the suggestion before you've done all of the work.
+If you're not familiar with GitHub, you can also submit a comment or question
+through one of our [communication channels][comms].
 
 For more details on the workflow of how to contribute via GitHub, see this
 community contributed [tutorial][git-tutorial].
 
-## Contribute to Lessons in the HPC Carpentry Incubator
-
-Any lesson that follows our [Code of Conduct][coc], and is licensed either [CC-BY][cc-by]
-or [CC-0][cc-0] can be listed in the HPC Carpentry Incubator. You can learn more
-about The Carpentries approach to curriculum development in the
-[Curriculum Development Handbook][cdh].
-
-The goal of the HPC Carpentry Incubator is to be a place for community members to
-share resources in early stages of development. The
-[Community Developed Lessons][community-lessons] page lists all lessons currently
-hosted in the HPC Carpentry Incubator. People already familiar with The
-Carpentries teaching practices can pick them up and teach them in meetups, in
-class, or in complement of a
-["standard" Carpentries 2-day workshop](https://carpentries.org/workshops/#workshop-core).
-The lessons can also be used by independent learners, outside of workshops.  
-
-We will also try to provide a friendly, community-supported, peer-review process for
-lessons.
-After the peer-review process, the lessons can be added to the
-HPC Carpentry Lab. At this time, we are not accepting lesson
-submissions to HPC Carpentry Lab. If you
-are interested in having a lesson reviewed for inclusion in the Lab, please submit
-it first to the Incubator first.
+## Contributing New Lessons
 
+Any lesson that follows the [Code of Conduct][coc] and is licensed either
+[CC-BY][cc-by] or [CC-0][cc-0] can be proposed for HPC Carpentry.
```

Comment:
So when I had left the incubator in place, I deliberately didn't say that it needed to follow the Carpentries lesson style so as to be more open. However, if these are being placed as new lessons then I think we should expect that they do.

Reply:
Addressed by 279e9729.